### ISSUE TITLE
Golden Lamb to Slaughter Cosmetic makes hexed unit untargetable

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Dota2-Gameplay#9898]
1.  Pick Shadow Shaman, equip golden lamb to slaughter cosmetic
2.  Hex target
3.  Attempt to target **back half of sheep** with spells/items/attacks
4. Get 'No Target' message

Affects teammates ability to target hexed unit too.

**Issue does not occur with Lamb to the Slaughter, or default Hex.**

https://github.com/ValveSoftware/Dota-2/assets/53882898/d1ccab70-ada1-41ae-bacf-fd36d2f891a2

Match ID : 7216637027 @45:10 can see muerta unable to kill witch doctor and just runs towards instead.
Happens multiple times throughout game.

**Comment by Astiberon** [comment]
There may already be an update that fixes this bug. Because I don't have this problem right now.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/53ae1a8d-79e0-43ae-9ae4-416dd39ae9fd)

**Comment by AMeseldzija** [comment]
Nah, its still an issue.
Only occurs when targeting the hind of the sheep.

https://github.com/ValveSoftware/Dota2-Gameplay/assets/53882898/dbca7c1c-8878-4deb-b288-d0abb2a173c2

**Comment by Astiberon** [comment]
I do not know what this bug is connected with, but I do not have this problem.
https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/29067ff0-90f5-41fc-8faa-1fe04beb0d4d

**Comment by AMeseldzija** [comment]
> I do not know what this bug is connected with, but I do not have this problem. https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/29067ff0-90f5-41fc-8faa-1fe04beb0d4d

Try target the back of the sheep.
https://github.com/ValveSoftware/Dota2-Gameplay/assets/53882898/0a31cb88-b4b8-4adf-babd-b8f5a273f7a9

Easiest to replicate on a horizontal plane. with the enemy facing to the right, but can replicate from any angle
https://github.com/ValveSoftware/Dota2-Gameplay/assets/53882898/f520a175-1ed1-4abc-b6a5-74423c2bd3e5

**Comment by Astiberon** [comment]
Yes you are right. The back of the sheep model really cannot be selected with the cursor.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/eb6c2064-a265-44ad-89f9-db9b5205d598)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107125193/0f14b2a6-7e46-4d80-99f2-61e4d38c1cd1)

**Comment by bizzozeron** [comment]
Fixed in future update. Your aim will no longer be blinded by their golden glory..

**Comment by Polykomodo** [comment]
> Fixed in future update. Your aim will no longer be blinded by their golden glory..
lul, mine was too greedy to damage gold #greedisgood
BTW thanks for your hard work!
